dist.java/java/404.html

```diff
@@ -38,10 +38,10 @@ <h4>Connected to <span ng-bind="containerName"></span></h4>
       <div ng-include src="viewPartial"></div>
     </div>
 
-    <script src="libs-2db3f78110.js"></script>
```

Comment:
Should the old JS libs-* and apps-* files be deleted?

Reply:
You are right. I've removed the old files.